src/main/java/com/coveros/selenified/element/check/Matches.java

```diff
@@ -187,22 +187,22 @@ default String checkValue(String expectedPattern, double waitFor, double timeToo
      * @param timeTook        - the amount of time it took for wait for something (assuming we had to wait)
      * @return String: the actual selected option of the element. null will be returned if the element isn't present or a select
      */
-    default String checkSelectedOption(String expectedPattern, double waitFor, double timeTook) {
+    String checkSelectedOption(String expectedPattern, double waitFor, double timeTook) {
         // record the action, and check it's a select
         if (!isPresentSelect(
-                getElement().prettyOutput() + " having a selected option to match a pattern " +
+                this.element.prettyOutput() + " having a selected option to match a pattern " +
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal " having a selected option to match a pattern " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)